pkg/standalone/run.go

```diff
@@ -213,6 +220,36 @@ func (config *RunConfig) Validate() error {
 	return nil
 }
 
+func (config *RunConfig) ValidateK8s() error {
```

Comment:
when RunConfig moves, this function will also move.

Reply:
I think we should put a description to this method or rename the method to somethings `validateRunConfigK8s` wdyt ?